README.rst

```diff
@@ -101,6 +101,15 @@ dependencies for the current platform by doing the following:
 On Windows, the virtualenv would be activated by executing
 ``env\Scripts\activate``.
 
+To run the unittests, you can execute the following.  Because of how
+nose searches for tests, specifying the directory is important as it
+will otherwise attempt to run non-unit tests as well (which will
+hang).
+
+.. code:: console
+
+    $ nosetest pyOCD/tests
+
```

Comment:
I'm going to add an example showing capturing code coverage since that is very useful.  You can capture and generate HTML coverage results by doing the following.  To enable this, I will also add the `coverage` package to the `dev-requirements.txt`:

```
nosetests --with-coverage --cover-html --cover-package=pyOCD pyOCD/test
```